packages/web-frontend/src/components/View/ChartWrapper.js

```diff
@@ -94,7 +100,13 @@ export const TABS = {
   TABLE: 'table',
 };
 
-export const ChartWrapper = ({ viewContent, isEnlarged, isExporting, onItemClick }) => {
+export const ChartWrapper = ({
+  viewContent,
+  isEnlarged,
+  isExporting,
```

Comment:
I think this needs a rework. isExporting made sense when it was just .png, now that there are two options we should rename these for clarity.

Maybe something like exportFormat = 'png' | 'pdf' | undefined. isExporting is defined on ui-components/src/components/Chart so we should make the change there.

Reply:
Yeah isPDFExporting is confusing.. We can use `exportFormat` in ChartWrapper, but I suggest we leave the logic here instead of in `Chart.js`, since they are only style component different.

Reply:
Ah so Chart renders the same if isExporting, but ChartWrapper renders differently for png and pdf? If so, your suggestion makes sense 👍